Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:52 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:52 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:52 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:52 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:52 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup  939524096 2019-03-21 05:53 /myfile1._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:53 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:53 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:53 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:53 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup  805306368 2019-03-21 05:55 /myfile1._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:55 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:55 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:55 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:55 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
2019-03-21 05:57:22,979 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.io.EOFException: End of File Exception between local host is: "clnode087.clemson.cloudlab.us/130.127.133.96"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
2019-03-21 05:57:22,988 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2674)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Trying to failover after sleeping for 896ms.
2019-03-21 05:57:23,890 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode087.clemson.cloudlab.us/130.127.133.96 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020 after 2 failover attempts. Trying to failover after sleeping for 2780ms.
2019-03-21 05:57:27,709 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 33 has reached the threshold 0.9990 of total blocks 33. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 33 has reached the threshold 0.9990 of total blocks 33. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 3 failover attempts. Retrying after sleeping for 4393ms.
2019-03-21 05:57:32,106 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 33 has reached the threshold 0.9990 of total blocks 33. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 3 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 33 has reached the threshold 0.9990 of total blocks 33. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 3 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 3 failover attempts. Retrying after sleeping for 10673ms.
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:57 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:57 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:57 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:57 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:56 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:59 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:58 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:59 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:58 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 05:58 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:01 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:00 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:01 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:00 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:00 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
2019-03-21 06:02:42,411 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1278ms.
2019-03-21 06:02:43,696 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode087.clemson.cloudlab.us/130.127.133.96 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 2313ms.
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:03 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:01 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:03 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:01 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:01 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:04 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:03 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:04 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:03 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:03 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:06 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:05 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:06 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:05 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:05 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:08 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:07 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:08 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:07 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:07 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
2019-03-21 06:08:40,504 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 938ms.
2019-03-21 06:08:41,450 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode087.clemson.cloudlab.us/130.127.133.96 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1735ms.
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
2019-03-21 06:09:07,880 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.io.IOException: DestHost:destPort node-1-link-0:8020 , LocalHost:localPort clnode087.clemson.cloudlab.us/130.127.133.96:0. Failed on local exception: java.io.IOException: Connection reset by peer, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1043ms.
2019-03-21 06:09:08,939 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 2288ms.
2019-03-21 06:09:11,230 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode087.clemson.cloudlab.us/130.127.133.96 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Trying to failover after sleeping for 2607ms.
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:09 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:08 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:09 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:08 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:09 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:11 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:10 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:11 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:10 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:10 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:13 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:12 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:13 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:12 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:12 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
2019-03-21 06:14:55,728 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.io.EOFException: End of File Exception between local host is: "clnode087.clemson.cloudlab.us/130.127.133.96"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
2019-03-21 06:14:55,737 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 16 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 16 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Retrying after sleeping for 1017ms.
2019-03-21 06:14:56,760 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 15 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 15 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Retrying after sleeping for 1556ms.
2019-03-21 06:14:58,320 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 13 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 13 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Retrying after sleeping for 4862ms.
2019-03-21 06:15:03,185 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Retrying after sleeping for 6881ms.
2019-03-21 06:15:10,070 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 2 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.analyzeFileState(FSDirWriteFileOp.java:591)
	at org.apache.hadoop.hdfs.server.namenode.FSDirWriteFileOp.validateAddBlock(FSDirWriteFileOp.java:172)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getAdditionalBlock(FSNamesystem.java:2679)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.addBlock(NameNodeRpcServer.java:875)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.addBlock(ClientNamenodeProtocolServerSideTranslatorPB.java:561)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot add block to /myfile3._COPYING_. Name node is in safe mode.
The reported blocks 24 has reached the threshold 0.9990 of total blocks 24. The number of live datanodes 4 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 2 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 14 more
, while invoking ClientNamenodeProtocolTranslatorPB.addBlock over node-0-link-0/10.10.1.1:8020 after 1 failover attempts. Retrying after sleeping for 17329ms.
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:15 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:15 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:15 /myfile3
-rw-r--r--   3 root supergroup  134217728 2019-03-21 06:15 /myfile4._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:13 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:17 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:17 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:17 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:17 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:17 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
diff succeed!
file 3 has been read from HDFS for 6 time
diff succeed!
file 3 has been read from HDFS for 7 time
diff succeed!
file 3 has been read from HDFS for 8 time
diff succeed!
file 3 has been read from HDFS for 9 time
diff succeed!
file 3 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile3
file 3 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:18 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:18 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:19 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:18 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-21 06:19 /myfile5
file 3 has been put into HDFS
file 3 has been read from HDFS for 1 time
diff succeed!
file 3 has been read from HDFS for 2 time
diff succeed!
file 3 has been read from HDFS for 3 time
diff succeed!
file 3 has been read from HDFS for 4 time
diff succeed!
file 3 has been read from HDFS for 5 time
